master/usage/ipv6.md

```diff
@@ -1,31 +1,158 @@
+    to the appropriate IPv6 address or `::` for all IPv6 addresses on
+	the host.
+  - The `--advertise-address` flag should be set to the IPv6 address that
+    nodes should use to access the kube-apiserver.
+  - The `--service-cluster-ip-range` flag should be set to an IPv6 CIDR
+    that will be used for the Service IPs, the DNS service address must
+	be in this range.
+- For kube-controller-manager:
+  - The `--master` flag should be set with the IPv6 address where the
+    kube-apiserver can be accessed.
+  - If the `--allocate-node-cidrs` flag is set then it is necessary to set
+    `--node-cidr-mask-size`. These are necessary when using host-local IPAM,
+    but if using calico-ipam is is easier to remove both flags as they are
+    not needed.
+  - The `--cluster-cidr` flag must be set to match the {{site.prodname}}
+    IPv6 IPPool.
+- For kube-scheduler:
+  - The `--master` flag should be set with the IPv6 address where the
+    kube-apiserver can be accessed.
+- For kubelet:
+  - The `--address` flag should be set to the appropriate IPv6 address
+    or `::` for all IPv6 addresses.
+  - The `--cluster-dns` flag should be set to the IPv6 address that will
+    be used for the service DNS, this must be in the range used for
+	`--service-cluster-ip-range`.
+  - The `--node-ip` flag should be set to the IPv6 address of the node.
+- For kube-proxy:
+  - The `--bind-address` flag should be set to the appropriate IPv6
+    address or `::` for all IPv6 addresses on the host.
+  - The `--master` flag should be set with the IPv6 address where the
+    kube-apiserver can be accessed.
+  - The `--cluster-cidr` flag must be set to match the {{site.prodname}}
+    IPv6 IPPool.
+
+### Limitations
+
+- Currently Kubernetes supports only one IP stack version at a time. This
+  means that if you configure Kubernetes for IPv6 then {{site.prodname}}
+  should be configured to only assign IPv6 addresses.
+- You should not switch a deployment that is using IPv4 to IPv6, or
+  vise-versa but to instead setup a new cluster.
+
+### Details of Enabling IPv6
+
+To enable IPv6 operation of a {{site.prodname}} deployment follow the
+steps below.
+
+1. Download the Calico manifest with the calico-config ConfigMap and
+   calico-node Daemonset as `calico.yaml`.
+1. Update the `calico.yaml` file with the following changes:
+   - If the ipam section of the `cni_network_config` has
+     `"type": "calico-ipam"` then it should be modified to have
+     [IPv4 assigning disabled and IPv6 assigning
+     enabled](/{{page.version}}/reference/cni-plugin/configuration#ipam).
+   - Add the following environment variables to the calico-node Daemonset
+
+     ```
+     - name: CALICO_IPV6POOL_CIDR
+       value: "fd20::0/112"
+     - name: IP6
+       value: "autodetect"
+     ```
+
+     - Set the `CALICO_IPV6POOL_CIDR` environment variable to the desired pool,
+       it should match the `--cluster-cidr` passed to the
+       kube-controller-manager and to kube-proxy.
```

Comment:
I do not know if you intended to add an additional level of nesting to this list with the indentation of line 113? I imagine this was just an error. If not, I would recommend against adding this extra level of nesting to the list.

Reply:
Yes it is an extra level of nesting because it applies to fields the above block is adding.